src/DropzoneArea.js

```diff
@@ -164,13 +164,13 @@ class DropzoneArea extends Component{
                     accept={this.props.acceptedFiles.join(',')}
                     onDrop={this.onDrop.bind(this)}
                     onDropRejected={this.handleDropRejected.bind(this)}
-                    className={classes.dropZone}
+                    className={classNames(classes.dropZone,this.props.dropZoneClass)}
                     acceptClassName={classes.stripes}
                     rejectClassName={classes.rejectStripes}
                     maxSize={this.props.maxFileSize}
                      >
                     <div className={classes.dropzoneTextStyle}>
-                        <p className={classes.dropzoneParagraph}>
+                        <p className={classNames(classes.dropzoneParagraph,this.props.dropzoneParagraphClass)}>
```

Comment:
These new props should be added to `DropzoneArea.propTypes` and `DropzoneArea.defaultProps`.